### PR TITLE
Network migrate secret agent to dasbus

### PR DESCRIPTION
**Pull request by rvykydal**
I am going to add unit tests for SecretAgent class.
If you want to see what actually changed better look at the first commit separately.

### CODE REVIEW

**Review by VladimirSlavik — DISMISSED** [review]
I can't comment (meaningfully) on the D-Bus stuff, the rest seems good to me. I wonder if it could have been easier to first split the dialog instead, but never mind.

File: pyanaconda/ui/gui/spokes/lib/network_secret_agent.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+            self.window.response(1)
+
+    def _update_connect_button(self):
+        self._connect_button.set_sensitive(self.valid)
+
+
+@dbus_interface(SECRET_AGENT.interface_name)
+class SecretAgent(object):
+    def __init__(self, ui_callback):
+        self._ui_callback = ui_callback
```
Comment (rvykydal) [inline pyanaconda/ui/gui/spokes/lib/network_secret_agent.py:125]: Add documentation for the callback.

**Review by poncovka — DISMISSED** [review]
It looks great. Thank you!

File: pyanaconda/ui/gui/spokes/lib/network_secret_agent.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/lib/network_secret_agent.py:134]: Done

**Review by poncovka — APPROVED** [review]

**Review by VladimirSlavik — APPROVED** [review]

**Comment by rvykydal** [comment]
I added unit tests in https://github.com/rhinstaller/anaconda/pull/2995/commits/2d1361fdff301f73e64e3ff3910c33e681f81980
and a small fixup in https://github.com/rhinstaller/anaconda/pull/2995/commits/06d18b4b60ae2ccee5900ea80d222381ccda5fa8 which I may squash with previous commit if you wish.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by poncovka — APPROVED** [review]

**Comment by rvykydal** [comment]
I've fixed placing of secret agent .glade file into wrong directory based on manual testing.

**Pull request merged**